auto-check-fix-commit/README.md

```diff
@@ -17,12 +17,19 @@ Reference - https://git-scm.com/docs/git-bisect/
 
 ### Usage
 
-1. Copy scripts `git-bisect-runner.sh` and `git-bisect-script.sh` to the directory of the project containing the fixed flaky test.
+1. Fork/clone this repo with the auto-check-fix-commit module containing the `git-bisect-runner.sh` and `git-bisect-script.sh` scripts.
 
+2. Within `git-bisect-runner.sh`, modify the `git bisect run ./git-bisect-script.sh $test_module $test_case` line to specify your path to `git-bisect-script.sh`, adding in `nohup` command at the beginning of the command to keep process running even after exiting shell/terminal.
 
-2. Run the script using the below command
+3. Within `git-bisect-script.sh`, modify the build line `mvn clean install -pl $test_module -am -DskipTests -Dlicense.skip` to be the build command for your project with the DeveloperFixed test. Also make sure that the NonDex version, within the script, is correct for your specific project.
```

Comment:
Can this also be achieved without modifying the script itself but just by passing in some additional arguments for the `mvn` command(s)? For example, could one run with `git-bisect-runner.sh --flaky ... --fixed ... --module ... -- test ... --mvn-install "-Dspotless.skip -Denforcer.skip"`.

Reply:
I believe it could. I can add: `'--mvn-install') shift; mvn_options="$1" ;;` after line 9 in `git-bisect-runner.sh` and then also in line 35 add at the end of the run command add `"$mvn_options"` for it to be: `git bisect run ./git-bisect-script.sh "$test_module" "$test_case" "$mvn_options"`.

And then in turn, `git-bisect-script.sh` would have to change mvn clean install line toI
`mvn clean install -pl $test_module -am -DskipTests -Dlicense.skip $mvn_options` and then also add `$mvn_options` to NonDex line. I can try testing this out on another branch.